### ISSUE TITLE
Active Cooling NuclearCraft Reactors Doesn't Work

### ISSUE THREAD

**Issue by DizzyDJW** [adam9899/Issues#608]
I've tried every possible combination of coolers, it doesn't seem as though any of them function.

I understand that you place an active fluid cooler inside next to the block to be cooled, I'm using Thermal Foundation fluiducts and resonant servos to transfer liquid from the buffer to the cooler, from the tank to the buffer and from the melter to the tank. The fluid coolers aren't taking in the liquid and the reactors refuse to process the additional cooling as valid causing the heat to increase rapidly. Any reason on why this is an issue only in this pack? I've been playing with NC for years and never had this issue before. I can add previews if need be.

**Comment by Avalonzo** [comment]
Please provide a screenshot of the inside of your reactor or if easier make your reactor setup using the [reactor planer](https://github.com/hellrage/NC-Reactor-Planner/releases/tag/v1.2.24) and attach the save file.

**Comment by DizzyDJW** [comment]
https://imgur.com/GKfzPLP

Added a bit of context and yes, the fluiducts are connected to a buffer each connected to ender tanks full of liquid enderium. I have literally everything connected up right and the reactor is VALID but the fluid coolers DO NOT WORK.

**Comment by Avalonzo** [comment]
I got it working after a bit of trial and error to use resonant ender as a coolant the active cooler needs to be in the corners then they'll act as enderium active fluid coolers see pics.

![LEU-235 5 x 3 x 5](https://user-images.githubusercontent.com/8142543/82105464-508fde00-9713-11ea-866f-aba7400f0af6.png)

![image](https://user-images.githubusercontent.com/8142543/82105482-67cecb80-9713-11ea-81ad-9b10ea7615d2.png)

**Comment by DizzyDJW** [comment]
Wow, so when the NC Reactor Planner states an "Active" version they don't mean to use that block with a fluid cooler adjacent to it, it means to use a fluid cooler as that block and pump the fluid into that block... I was so confused cuz the Copy Paste Gadget doesn't paste in anything where you place any "Active" blocks, and literally nobody expands upon Active Cooling in any tutorial, even Direwolf20 durped and messed up his 2019 tutorial when he mentions it for a moment. Thank you tons man!!

**Comment by Avalonzo** [comment]
You're most welcome and I did try your exact setup first but as you found out it didn't work so I looked at the a wiki or two but nowhere mentioned anywhere how to actually make enderium fluid coolers. So I suddenly thought to myself if your pumping resonant ender into an active cooler then maybe it needs to be in the corner that's where the enderium type coolers need to be and then it was like an epiphany. So now designing a bigger, better and safer reactor will be so much easier now we understand how active fluid coolers actually work.
